### PR TITLE
Add selenium based test

### PR DESCRIPTION
**Pull request by ankitjavalkar**
Need to modify the Test Quiz and questions based on the final Demo Quiz

### CODE REVIEW

File: yaksh/selenium_test/selenium_test.py [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+    selenium_test = SeleniumTest(url=url, quiz_name=quiz_name)
+    return selenium_test.run_load_test(*args)
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument('start', type=int, help="Starting user id")
+    parser.add_argument("-n", "--number", type=int, default=10, help="number of users")
+    opts = parser.parse_args()
+
+    url = "http://yaksh.fossee.aero.iitb.ac.in/exam/"
```
Comment (prabhuramachandran) [inline yaksh/selenium_test/selenium_test.py]: Can you make the URL something that can be passed from the command line with a sane default?

**Comment by prabhuramachandran** [comment]
The test is not exercised on travis, please fix.

File: yaksh/selenium_test/selenium_test.py [hunk 2]
```diff
@@ -0,0 +1,141 @@
+from selenium import webdriver
```
Comment (prabhuramachandran) [inline yaksh/selenium_test/selenium_test.py]: This can be just added to the tests directory instead of creating a new `selenium_test`.

**Comment by ankitjavalkar** [comment]
Closing until further updates

**Pull request closed without merging**